llvm/test/tools/llvm-objcopy/ELF/gap-fill.test

```diff
@@ -1,39 +1,32 @@
-# RUN: yaml2obj %s > %t
+# RUN: yaml2obj %s -o %t
 
-## Verify section headers before we perform several testings.
-# RUN: llvm-readelf -S %t | FileCheck %s --check-prefix=ORG-SHDR
-# ORG-SHDR: Section Headers:
-# ORG-SHDR:  [Nr] Name              Type            Address          Off    Size   ES Flg Lk Inf Al
-# ORG-SHDR:  [ 0]                   NULL            0000000000000000 000000 000000 00      0   0  0
-# ORG-SHDR:  [ 1] .nogap            PROGBITS        0000000000000102 000042 000006 00   A  0   0  1
-# ORG-SHDR:  [ 2] .gap1             PROGBITS        0000000000000108 000048 000007 00  AX  0   0  1
-# ORG-SHDR:  [ 3] .gap2             PROGBITS        0000000000000110 000050 000004 00   A  0   0  1
-# ORG-SHDR:  [ 4] .nobit_tbss       NOBITS          0000000000000180 000058 000018 00 WAT  0   0  8
-# ORG-SHDR:  [ 5] .foo              PROGBITS        0000000000000184 00005c 000004 00  WA  0   0  1
-# ORG-SHDR:  [ 6] .nobit_bss        NOBITS          000000000000018a 000060 000008 00  WA  0   0  1
+## This test is partially based on one from D67689.
```

Comment:
This reference to "D67689" is going to get archaic very quickly given we've moved to GitHub PRs and references to DXXXXX will be lost on newer contributors. If you feel it's important, you can add a note in the commit message (but please use the full URL, so that it can be followed).

Reply:
I have removed the reference.